src/main/java/com/nitnelave/CreeperHeal/block/CreeperBisected.java

```diff
@@ -0,0 +1,61 @@
+package com.nitnelave.CreeperHeal.block;
+
+import org.bukkit.block.BlockFace;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.data.Bisected;
+import org.bukkit.block.data.BlockData;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Bisected implementation of CreeperMultiblock.
```

Comment:
Can you add a bit of text about what is a "Bisected"?

Reply:
Sure. I figured most people looking to work on CH are probably somewhat familiar with [Bukkit's API](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/data/Bisected.html). I do note that Spigot's description does not include that stairs and slabs are also bisected while not being 2-block arrangements.